config/core/test-configs.yaml

```diff
@@ -1650,6 +1657,20 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: asus-C523NA-A20057-coral
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - cros-ec
+      - igt-gpu-i915
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - ltp-crypto
```

Comment:
Tested OK https://staging.kernelci.org/test/plan/id/60a5ae7de52d351427a35858/
Although all tests were skipped, most likely due to crypto configs not being turned on.